src/users/users.controller.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  HttpException,
+  HttpStatus,
+  NotFoundException,
+  Param,
+  Patch,
+  Post,
+  Query,
+} from '@nestjs/common';
+import { UsersService } from './users.service';
+import { User } from '@prisma/client';
+
+@Controller('users')
+export class UsersController {
+  constructor(private readonly usersService: UsersService) {}
+
+  @Get() //   GET /users or /users?min-links=value
```

Comment:
**praise**: Implementing query parameters for filtering users is a nice touch for flexibility.